core/templates/dev/head/pages/exploration_player/ConversationSkinDirective.js

```diff
@@ -615,11 +617,13 @@ oppia.directive('conversationSkin', [
             $scope.startCardChangeAnimation = true;
 
             $timeout(function() {
-              var newInteractionHtml = oppiaPlayerService.getInteractionHtml(
+              var newInteractionHtml =
+		ExplorationPlayerService.getInteractionHtml(
```

Comment:
Sorry for not catching it last time, please indent this line once from `var`, like this: 

![indent2](https://user-images.githubusercontent.com/7807557/30772892-4ab68206-a083-11e7-9489-bda4b315d368.jpeg)